music/src/core/recorder.ts

```diff
@@ -135,7 +137,7 @@ export class Recorder {
   }
 
   private midiReady(midi: WebMidi.MIDIAccess) {
-    console.log('Initialized Recorder');
+    logging.log('Initialized Recorder');
```

Comment:
I would use "Recorder" as the prefix here (and everywhere where the prefix could be helpful)